lib/datadog/appsec/contrib/rack/request.rb

```diff
@@ -45,6 +45,10 @@ def self.url(request)
           def self.cookies(request)
             request.cookies
           end
+
+          def self.form_hash(request)
```

Comment:
Can you add a comment about what type of object we expect to be returned here from this function.

Reply:
Sure (FYI `Hash<String,String>`)